templates/federated_reporting/10-base_filter.sed

```diff
@@ -14,7 +14,6 @@ s/public\.//g;
 
 # don't reset the promiselog sequence value
 /SELECT pg_catalog.setval('__promiselog_id_seq.*$/d
-/SELECT pg_catalog.setval('__lastseenhostslogs_id_seq.*$/d
```

Comment:
@craigcomstock is this safe to remove without care of the version of the binaries being run?

Reply:
We haven't yet release a version so this should be fine.